StitchCoreServicesAwsSes/Sources/StitchCoreServicesAwsSes/AwsSesSendResult.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/**
+ * The result of an AWS SES send request.
+ */
+public struct AwsSesSendResult: Decodable {
+    public enum CodingKeys: String, CodingKey {
```

Comment:
[minor] this is unnecessary if the keys are the same. We're better off having less code for these simple types